features/step_definitions/logging.rb

```diff
@@ -1078,13 +1078,11 @@
     cm_patch << ["p", "USERNAME=#{username}"] << ["p", "PASSWORD=#{password}"]
   end
 
-  if version == "6.8"
-    # get the arch of node
-    @result = admin.cli_exec(:get, resource: "nodes", l: "kubernetes.io/os=linux", output: "jsonpath={.items[0].status.nodeInfo.architecture}")
-    # set xpack.ml.enable to false when testing ES 6.8 on arm64 cluster
-    if @result[:response] == "arm64"
-      cm_patch << ["p", "MACHINE_LEARNING=false"]
-    end
+  # get the arch of node
+  @result = admin.cli_exec(:get, resource: "nodes", l: "kubernetes.io/os=linux", output: "jsonpath={.items[0].status.nodeInfo.architecture}")
+  # set xpack.ml.enable to false when testing ES 6.8 on arm64 cluster
```

Comment:
Looks like the comment needs update to make it in sync with the code changes.

Reply:
Yes, I'll update it. Thanks.